tests/all/test_passes.rs

```diff
@@ -12,6 +12,8 @@ fn test_init_all_passes_for_module() {
 
     pass_manager.add_argument_promotion_pass();
     pass_manager.add_constant_merge_pass();
+    #[cfg(any(feature = "llvm10-0", feature = "llvm11-0", feature = "llvm12-0"))]
```

Comment:
It's annoying but I'd rather invert the logic here so that it's future proofed. IE `#[cfg(not(any(feature = "llvm3-6", ..., feature = "9-0")))]`

Reply:
No worries, makes sense to pay the annoyance once up front instead of making you add another clause for each future version of LLVM.  I've made the edit and pushed it.